src/HttpCache/VarnishPurger.php

```diff
@@ -48,10 +48,10 @@ public function purge(array $iris)
             return sprintf('(^|\,)%s($|\,)', preg_quote($iri));
         }, $iris);
 
-        $regex = isset($parts[1]) ? sprintf('(%s)', implode(')|(', $parts)) : $parts[0];
+        $regex = count($parts) > 1 ? sprintf('(%s)', implode(')|(', $parts)) : array_shift($parts);
```

Comment:
No because the array has the following format: `['/foo' => '/foo', '/bar' => '/bar']`